### PR TITLE
Send Eichrecht Meter public Key via DataTransfer Request

### PR DESCRIPTION
**Pull request by abithgs**

### CODE REVIEW

**Review by FFrancieli — APPROVED** [review]

**Comment by MounirKaroudEV** [comment]
Maybe in the future we will have to add the EVSE also, since you can have multiple connectors with same id

**Review by MounirKaroudEV — APPROVED** [review]

File: simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/PayloadFactory.java [hunk 1]
```diff
@@ -209,4 +218,31 @@ private SignedMeterValue createSignedMeterValues(String stationId, TransactionEv
         }
         return null;
     }
+
+    DataTransferRequest createPublicKeyDataTransfer(List<Integer> connectorIds) {
```
Comment (akoshterek) [inline simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/PayloadFactory.java:222]: Does it make sense to make it per connector? Meters are per EVSE.

Moreover, in sendEichrechtMeterPublicKeys you will call this method with the same arguments for multi-EVSE configurations.

Comment (abithgs) [reply simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/PayloadFactory.java:222]: Will change to per Evse. It makes more sense here

File: simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/PayloadFactory.java [hunk 2]
```diff
@@ -209,4 +218,31 @@ private SignedMeterValue createSignedMeterValues(String stationId, TransactionEv
...
+    }
+
+    @Getter
+    public static class MeterConfig {
+        int connectorId;
+        String meterSerial;
+        String type;
+        String publicKey;
+
+        public MeterConfig(int connectorId) {
```
Comment (akoshterek) [inline simulator-core/src/main/java/com/evbox/everon/ocpp/simulator/station/PayloadFactory.java:241]: should be evseId

**Review by akoshterek — APPROVED** [review]

**Pull request merged**